### PR TITLE
Fix continuous tests by breaking it down

### PR DESCRIPTION
**Pull request by gargnitingoogle**
### Description
Fixes the issue in internal bug 272656921 .

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - Tested manually by running [run_load_test_and_fetch_metrics.sh](https://github.com/GoogleCloudPlatform/gcsfuse/pull/1432/files#diff-ff86a10afec13e1928d1cf8ff4f5c5c5fe7aed9794cd54e6931d9509e1b96643) in a VM. In sequence, all 16 jobs started, executed, logged their start, end times and their fio outputs, in individual files and then their outputs were analyzed by fetch_metrics.py one after another. 

**They have not been manually tested for upload yet. **

2. Unit tests - NA
3. Integration tests - NA

### CODE REVIEW

**Review by Tulsishah — COMMENTED** [review]
Please mention manual testing details in description.

**Review by sethiay — COMMENTED** [review]
Also, the base branch should be master.

File: perfmetrics/scripts/run_load_test_and_fetch_metrics.sh [hunk 1]
```diff
@@ -21,18 +21,33 @@ sudo apt-get install pip -y
...
+
+# Run individual jobs for each .fio job file in job_files/seq_rand_read_write/*_thread.fio.
+# Create a separate fio-output json file for each, record their names 
+# in an array, and calculate metrics and upload each individually.
+fi_output_jsons=()
 echo "Overall fio start epoch time:" `date +%s`
-fio job_files/seq_rand_read_write.fio --lat_percentiles 1 --output-format=json --output='fio-output.json'
+for job_group_path in job_files/seq_rand_read_write/*_thread.fio; do
+  job_group_name=$(basename "$job_group_path")
+  echo "Overall fio start epoch time for "${job_group_name}" :" `date +%s`
```
[the remark this answers is not in the document]
Comment (gargnitingoogle) [reply perfmetrics/scripts/run_load_test_and_fetch_metrics.sh]: It's the overall time for that job-group which has 40 parallel jobs. But I have removed the word overall from there.

File: perfmetrics/scripts/job_files/seq_rand_read_write/glob-include.fio [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+direct=1
+fadvise_hint=0
+verify=0
+rw=read
+bs=1M
+iodepth=64
+invalidate=1
+ramp_time=10s
+runtime=60s
+startdelay=2m
```
[the remark this answers is not in the document]
Comment (gargnitingoogle) [reply perfmetrics/scripts/job_files/seq_rand_read_write/glob-include.fio]: Done.

File: perfmetrics/scripts/run_load_test_and_fetch_metrics.sh [hunk 3]
```diff
@@ -21,18 +21,33 @@ sudo apt-get install pip -y
 echo Print the time when FIO tests start
 date
 echo Running fio test..
+
+# Run individual jobs for each .fio job file in job_files/seq_rand_read_write/*_thread.fio.
+# Create a separate fio-output json file for each, record their names 
+# in an array, and calculate metrics and upload each individually.
+fi_output_jsons=()
```
[the remark this answers is not in the document]
Comment (gargnitingoogle) [reply perfmetrics/scripts/run_load_test_and_fetch_metrics.sh]: Good catch. Done.

**Comment by gargnitingoogle** [comment]
> Please mention manual testing details in description.

Added details. Upload to gsheet has not been tested yet. Will test that and add.

**Comment by gargnitingoogle** [comment]
> Also, the base branch should be master.

Done.

File: perfmetrics/scripts/run_load_test_and_fetch_metrics.sh [hunk 4]
```diff
@@ -21,18 +21,33 @@ sudo apt-get install pip -y
...
-then
-  python3 fetch_metrics.py fio-output.json
-else
-  python3 fetch_metrics.py fio-output.json --upload
+if [ "${KOKORO_JOB_TYPE}" == "RELEASE" ] || [ "${KOKORO_JOB_TYPE}" == "CONTINUOUS_INTEGRATION" ] || [ "${KOKORO_JOB_TYPE}" == "PRESUBMIT_GITHUB" ]; then
+  ARGS="--upload"
 fi
+
+for fio_output_json in ${fi_output_jsons[@]}: do
+  python3 fetch_metrics.py ${fio_output_json} $ARGS
```
[the remark this answers is not in the document]
Comment (gargnitingoogle) [reply perfmetrics/scripts/run_load_test_and_fetch_metrics.sh]: @sethiay I am not sure how to test the upload part. I don't have the right permissions/accesses for this. Let's connect offline on this.

**Comment by gargnitingoogle** [comment]
This is an outdated PR for a task which has been completed by an alternate approach. So, this PR is no longer needed. Closing it.

**Pull request closed without merging**